contents/docs/error-tracking/cutting-costs.mdx

```diff
@@ -24,6 +24,35 @@ Alternatively, you can disable exception autocapture completely in your [project
 
 ## Filtering exceptions clientside
 
+### Suppression rules
+
+Autocaptured exceptions can be ignored client-side by [configuring suppression rules](https://us.posthog.com/error_tracking/configuration?tab=error-tracking-exception-autocapture#selectedSetting=error-tracking-exception-autocapture).
+Given the stack of an exception may still be minified client-side you are only permitted to filter based on the exception type and message attributes.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/suppression_rule_light_8db44eb6b1.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/suppression_rule_dark_5972d57398.png"
+    alt="Issue suppression rules"
+    classes="rounded"
+/>
+
+### Burst protection
+
+The PostHog JS SDK uses burst protection to limit the number of autocaptured exceptions that can be captured in a period.
+This prevents an excessive amount of exceptions being captured from any one client, typically because they're being thrown in an infinite loop.
+By default we capture 10 exceptions (bucket size) of the same type in a 10 second period before the rate limiter kicks in.
+After which we capture 1 exception (refill rate) every 10 seconds.
+Often unnecessary unless advised, the bucket size and refill rate can be edited as part of your configuration:
```

Comment:
Assuming this is PostHog JS config?
```suggestion
The PostHog JS SDK uses burst protection to limit the number of autocaptured exceptions that can be captured in a period. This prevents an excessive amount of exceptions being captured from any one client, typically because they're being thrown in an infinite loop.

By default, we capture 10 exceptions (bucket size) of the same type in a 10 second period before the rate limiter kicks in, after which, we capture 1 exception (refill rate) every 10 seconds.

Often unnecessary unless advised, the bucket size and refill rate can be edited as part of your configuration:
```